src/commands/create.js

```diff
@@ -2,21 +2,21 @@ const { flags: flagsHelper } = require('@oclif/command');
 const fs = require('fs');
 const path = require('path');
 const inquirer = require('inquirer');
-const got = require('got');
+const httpRequest = require('@apify/http-request');
```

Comment:
I'm for use directly got directly. The http-request is using got underhood anyway.